dpareto/data/adult/downloader.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pickle
+import mxnet as mx
+from mxnet import nd
+import urllib.request
+import shutil
+import ssl
+
+print("Downloading...")
+ssl._create_default_https_context = ssl._create_unverified_context
+training_data_url = 'https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a1a.t'
+testing_data_url = 'https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a1a'
```

Comment:
Maybe worth pulling these urls out into a text file in case of the dataset being moved??